Anabi/Features/Assets/ModifyAssetHandler.cs

```diff
@@ -22,11 +23,25 @@ public async Task<MinimalAssetViewModel> Handle(ModifyMinimalAssetObj assetObj)
             var message = assetObj.ModifyMinimalAsset;
             var id = assetObj.Id;
             var assetEntity = await context.FindAsync<AssetDb>(id);
+            
+            
+            var historicalStageDb = new HistoricalStageDb()
```

Comment:
Please update the historical stage that comes from the GET asset minimal details for the asset, do not add a new one (i know i said something else, but i realized it is not correct now)

Reply:
Nevermind I found the issue. When I retrieve the assetEntity here, this is retrieved without the HistoricalStages.
````
var assetEntity = await context.FindAsync<AssetDb>(id);
````